--- yandextank-1.7.12-orig/yandextank/plugins/ResourceCheck.py	2015-02-16 18:31:14.000000000 -0500
+++ yandextank-1.7.12/yandextank/plugins/ResourceCheck.py	2015-08-02 07:15:32.560086686 -0400
@@ -21,8 +21,8 @@
         '''         Constructor        '''
         AbstractPlugin.__init__(self, core)
         self.interval = "10s"
-        self.disk_limit = 2048  # 2 GB
-        self.mem_limit = 512  # 0.5 GB
+        self.disk_limit = 512  # 0.50 GB
+        self.mem_limit = 256   # 0.25 GB
         self.last_check = 0
 
     def get_available_options(self):
